packages/core/src/lib/TableNew/atoms/Pagination.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ITableNewProps } from '../types';
+import styles from './styles';
+import { paginate } from '../Helper';
+
+const { PaginationStyled, PaginationTag, PaginationText } = styles;
+
+type TTableProps = Pick<
+    ITableNewProps,
+    | 'noPagination'
+    | 'maxPages'
+    | 'customPageNumber'
+    | 'onPageNumberChanged'
+    | 'pageSize'
+>;
+interface IPaginationProps extends TTableProps {
+    pageNum: number;
+    tableData: (React.ReactNode | string)[][];
+    handleSetPageNumber: (pageNum: number) => void;
+}
+
+const Pagination: React.FC<IPaginationProps> = ({
```

Comment:
Please use the existing pagination component.
folder path -> `core/src/lib/pagination`